cli/src/main.rs

```diff
@@ -42,12 +42,19 @@ struct TestArgs {
 struct ValidateArgs {
     #[arg(
         long,
-        required = true,
+        required_unless_present = "bazel_bep_path",
+        conflicts_with = "bazel_bep_path",
```

Comment:
This allows it to be mutually exclusive with the `--junit-paths` as well as `--xc-result-path` when applicable